src/modules/enrollment/components/pages/EnrollmentLocationMapPage.tsx

```diff
@@ -0,0 +1,26 @@
+import { Box } from '@mui/material';
+import TitleCard from '@/components/elements/TitleCard';
+import NotFound from '@/components/pages/NotFound';
+import EnrollmentLocationMap from '@/modules/enrollment/components/EnrollmentLocationMap';
+import useEnrollmentDashboardContext from '@/modules/enrollment/hooks/useEnrollmentDashboardContext';
+
+import { clientBriefName } from '@/modules/hmis/hmisUtil';
+
+const EnrollmentLocationMapPage = () => {
+  const { enrollment } = useEnrollmentDashboardContext();
+
+  if (!enrollment) return <NotFound />;
+
+  return (
+    <TitleCard title='Location Map' headerVariant='border'>
+      <Box sx={{ p: 2 }}>
```

Comment:
minor thing, I might take the padding off to make better use of horizontal space on mobile

Reply:
good idea, updated!